main/main.gd

```diff
@@ -77,6 +80,8 @@ func create_players(num):
 		player.update.connect(_on_update)
 		player.sprite.texture = sprites[i]
 		player.sprite.visibility_layer = 9
+		
+	State.queue = State.players.duplicate()
```

Comment:
Setting up the queue requires too much knowledge of the inner working of the state object for my liking. I prefer a small abstraction for this but not a significant concern tbh.

Reply:
done